### ISSUE TITLE
Enlaces rotos en Recetario - Wiki PyAr

### ISSUE THREAD

**Issue by cabustillo13** [PyAr/wiki#122]
# Estructura de datos
* **IterarSobrePares** -> No tiene ningún link asociado a ese título.

# Formatos, datos, números y conversiones
* **aLetras** -> Al darle click te redirige a http://www.python.org.ar/wiki/Recetario.
* **Reverse** -> Al darle click te redirige a http://www.python.org.ar/wiki/Recetario.
* **ValidarCuit** -> Devuelve " _Movido a Recetario/ValidarCuit (eliminar esta página en cuanto sea posible)_"
* **CalcularDigitoVerificadorModuloDiez** -> Devuelve "_This page doesn't exist yet._"
* **NormalizarCaracteresUnicode** ->  Devuelve "_This page doesn't exist yet._"

# Frameworks Web
* **Hola Mundo** -> Error 404.

# Interfaces graficas
## Gtk
* **/Gui/Gtk/HolaMundo** -> Devuelve "_None_"
* **/Gui/Gtk/HolaMundoOO** -> Devuelve "_None_"
* **/Gui/Gtk/Entry** -> No muestra nada.
* **/Gui/Gtk/EntrySoloNumeros** -> No muestra nada.
* **/Gui/Gtk/HBox** -> No muestra nada.
* **/Gui/Gtk/VBox** -> No muestra nada.
* **/Gui/Gtk/Grid** -> No muestra nada.
* **/Gui/Gtk/Button** -> No muestra nada.
* **/Gui/Gtk/ButtonBox** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/AutoComplete** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/Dialog** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/FileChooser** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/Menu** -> Devuelve "_None_"
* **/Gui/Gtk/TextArea** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/PrintNonGtk** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/RichText** -> No muestra nada.
* **/Gui/Gtk/ConfirmClose** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/MultiThread** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/MultiThread2** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/Runner** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/FuncionRunner** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/LabelConColor** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/XMLRPCServer** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/StockItems** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/WebkitEditor** -> Devuelve "_None_"
* **/Gui/Gtk/StatusIcon** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/ErrorHandler** ->  Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/ListView** -> Devuelve "_This page doesn't exist yet._"
* **/Gui/Gtk/EmuladorTerminal** -> Devuelve "_This page doesn't exist yet._"

## Gtk + glade
* **GtkGladeHolaMundoOO** -> No tiene ningún link asociado a ese título.
* **/GtkGladeSignals** -> Devuelve "_This page doesn't exist yet._"

## Qt
* **/QtImprimirPagina** -> No muestra nada.
* **/QtExtraerTextoRecurso** -> Devuelve "_This page doesn't exist yet._"

# Emails
* **/GmailMail** -> Devuelve "_This page doesn't exist yet._"
* **/EmailConAdjunto** -> Devuelve "_This page doesn't exist yet._"

# Creación de ejecutables para Windows
* **/CrearEjecutableWindowsDesdeLinux** -> Devuelve "_This page doesn't exist yet._"
* **/CrearEjecutableWindows** -> Devuelve "_None_"

# XML
* **SimpleXmlElement** -> Al darle click te redirige a http://www.python.org.ar/wiki/Recetario.

# Pdf
* **/FacturaPyFpdf** -> No devuelve nada.
* **/EstilosRst2Pdf** -> Devuelve "_This page doesn't exist yet._"

# Dbf
* **/DbfPy** -> Devuelve "_This page doesn't exist yet._"

# Windows
* **/ServidorCom** -> Devuelve "_This page doesn't exist yet._"
* **/WinBatt** -> Devuelve "_This page doesn't exist yet._"

# Numpy, Scipy, Matplotlib
* **/Histograma** -> Devuelve "_None_"

# Administracion de Sistemas Operativos
* **/ListarProcesos** -> Devuelve "_This page doesn't exist yet._"
* **/ChequearInterfacesInternetLinux** -> Devuelve "_This page doesn't exist yet._"

# Internet
* **/ObtenerBytesTransferidos** -> Devuelve "_This page doesn't exist yet._"
* **/ObtenerUbicacionGeografica** -> Devuelve "_This page doesn't exist yet._"
* **/RevisarConexion** -> Devuelve "_This page doesn't exist yet._"

# Misceláneo
* Todos los link acá devuelven "_This page doesn't exist yet._"

# Python en Apache OpenOffice / LibreOffice
* **/pyUNO/MiPrimerMacro** -> Devuelve "_This page doesn't exist yet._"

# Crypto
* **/Crypto/BlowfishConBlowfishpy** -> Devuelve "_This page doesn't exist yet._"

# Multiprocessing y threading (y otras yerbas)
* **/MultiprocessingYThreading:** -> Devuelve "_This page doesn't exist yet._"

# Divertidos
* **/Fun/NadoSincronizado** -> Devuelve "_This page doesn't exist yet._"
* **/Fun/NadoSincronizadoDisco** -> Devuelve "_None_"
* **/Fun/MiniSpaceInvaders** -> Devuelve "_This page doesn't exist yet._"

# Al final de la página devuelve:

*CategoryRecetas*

*System Message: WARNING/2 (<string> line 396) id1*

*Duplicate explicit target name: "autocompletado en consola interactiva".*

**Comment by tzulberti** [comment]
Hola.

Muchas gracias por cargar el issue. Puede ser que esto este pasando en http://www.python.org.ar/wiki/Recetario?

Lo comento porque estuvimos trabajando en una nueva wiki: (https://wiki.python.org.ar/recetario), donde corregimos un monton de estos temas. Todavía no la sacamos a producción pero la idea es hacerlo en un par de semanas

**Comment by cabustillo13** [comment]
Sí, esos issues son del link viejo entonces http://www.python.org.ar/wiki/Recetario .
